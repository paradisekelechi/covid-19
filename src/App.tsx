import React from 'react';
import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
import { createGlobalStyle } from 'styled-components';
import { Page } from './components';

const GlobalStyle = createGlobalStyle`
  @import url('https://fonts.googleapis.com/css?family=Berkshire+Swash|Kodchasan:400,600,700&display=swap');
  body {
    width: 100%;
    height: 100%;
    padding: 0;
    margin: 0;
    font-family: 'Kodchasan';
  }
`;

function App() {
  return (
    <Router>
      <GlobalStyle />
      <Switch>
        <Route path="/" component={Page} />
      </Switch>
    </Router>
  );
}

export default App;
