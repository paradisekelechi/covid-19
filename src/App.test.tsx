import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import App from './App';

describe('App', () => {
  test('renders properly', () => {
    const wrapper = shallow(<App />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
})