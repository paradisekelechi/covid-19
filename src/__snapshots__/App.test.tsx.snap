// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders properly 1`] = `
<BrowserRouter>
  <GlobalStyleComponent />
  <Switch>
    <Route
      component={[Function]}
      path="/"
    />
  </Switch>
</BrowserRouter>
`;
